src/abstract.js

```diff
@@ -242,12 +242,12 @@ Sk.abstr.numOpAndPromote = function (a, b, opfn) {
     }
 
     if (a.constructor === Sk.builtin.lng) {
-//      if (b.constructor == Sk.builtin.nmber)
-//          if (b.skType == Sk.builtin.nmber.float$) {
-//              var tmp = new Sk.builtin.nmber(a.tp$str(), Sk.builtin.nmber.float$);
-//              return [tmp, b];
-//          } else
-//              return [a, b.v];
+        //if (b.constructor == Sk.builtin.nmber)
```

Comment:
I did these changes because the build server was not giving green light. I can revert them easily because they are in separate commits.